lib/ssl/test/ssl_cert_SUITE.erl

```diff
@@ -222,7 +222,8 @@ all_version_tests() ->
      extended_key_usage_client_auth,
      cert_expired,
      no_auth_key_identifier_ext,
-     no_auth_key_identifier_ext_keyEncipherment
+     no_auth_key_identifier_ext_keyEncipherment,
+     duplicate_chain
```

Comment:
I guess you added the test earlier but forgot to list it?!

Reply:
Also true, found this by strange coverage when I ran test suite.